
make[2]: *** No rule to make target `/Volumes/btc-0.11/build/glibc-m68k-linux-gnu-3/ld.map', needed by `/Volumes/btc-0.11/build/glibc-m68k-linux-gnu-3/elf/ld.so'.  Stop.
make[2]: Leaving directory `/Volumes/btc-0.11/build/glibc-2.10.1/elf'
make[1]: *** [elf/subdir_lib] Error 2
make[1]: Leaving directory `/Volumes/btc-0.11/build/glibc-2.10.1'
make: *** [all] Error 2

From glibc git commit 0079dd2382eba185d22410a1d3088dc7146bc00a.
(eglibc svn revision 9281).

It is needed to avoid the above error when backporting m68k TLS calls.

diff --git a/Versions.def b/Versions.def
index a8a9cd2..70de1c1 100644
--- a/Versions.def
+++ b/Versions.def
@@ -28,6 +28,7 @@ libc {
   GLIBC_2.9
   GLIBC_2.10
   GLIBC_2.11
+  GLIBC_2.12
 %ifdef USE_IN_LIBIO
   HURD_CTHREADS_0.3
 %endif
